### PR TITLE
add harikanani profile

### PR DESCRIPTION
**Pull request by harikanani**
Added my profile that resolves issue number #469

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -814,5 +814,11 @@
...
     "imageUrl": "https://user-images.githubusercontent.com/29454668/115579668-f8853280-a2f8-11eb-8372-95202883ac86.png",
     "issueNumber": 451
-  }
+  }, 
+  {
+    "name": "Harikrushn",
+    "githubUsername": "harikanani",
+    "imageUrl": "https://user-images.githubusercontent.com/54361799/116959271-c1306180-acba-11eb-8f59-2ee53b9da174.jpg",
+    "issueNumber": 469
+  },
```
Comment (schmelto) [inline data.json:823]:
```suggestion
  }
```

**Review by schmelto — CHANGES_REQUESTED** [review]
Looking good so far. Maybe one little change 😄

**Review by harikanani — COMMENTED** [review]
i have done all requested changes

**Review by harikanani — COMMENTED** [review]
resolved issue

**Comment by schmelto** [comment]
@AllanRegush do you know why the Linter is failing?

**Review by schmelto — APPROVED** [review]
PR is looking good for me. I wait for @AllanRegush maybe he or someone else got an idea because of the failing Linter

**Comment by harikanani** [comment]
> @AllanRegush do you know why the Linter is failing?

No i don't know why it's failing but I think now it's looks fine! Let's wait for @AllanRegush review.

**Comment by schmelto** [comment]
> No i don't know why it's failing but I think now it's looks fine! Let's wait for @AllanRegush review.

I've opened a PR for this on #478 but dont know where the linter error comes from.

**Comment by schmelto** [comment]
prettier was fixed I merged the PR thanks for the patience 🚀

**Pull request merged**